conformance-tests/build.gradle

```diff
@@ -68,8 +68,8 @@ distributions {
 }
 
 dependencies {
-    assertionsImplementation "org.jspecify:jspecify:0.0.0-SNAPSHOT"
-    depsImplementation "org.jspecify:jspecify:0.0.0-SNAPSHOT"
+    assertionsImplementation project(":")
```

Comment:
I would find a comment helpful here. I did a quick search and didn't find what `project(":")` means.

Reply:
The `project(…)` notation takes the "project path" to the depended-on project, and because the JSpecify "prod code" is in the root project, its path is `:`: https://docs.gradle.org/current/userguide/declaring_dependencies.html#sub:project_dependencies / https://docs.gradle.org/current/userguide/intro_multi_project_builds.html#sec:project_path

Alternatively, one can also directly pass a project instance, so this should work too but is less idiomatic:
```groovy
    assertionsImplementation rootProject
```

Reply:
I would find `rootProject` much more self-explanatory, but if `project(":")` is the more idiomatic way, that's also fine with me.

Reply:
Even more _idiomatic_ would be to move the code to a subproject (leaving the root project empty) so you'd use something like `project(":lib")` here :wink:

Reply:
I've added a comment, and I've included the recommendation that we redo the project structure.